root/artist/ArtistMerge.js

```diff
@@ -69,8 +69,23 @@ const ArtistMerge = ({$c, form, toMerge}: Props) => {
 
           <FormRowCheckbox
             field={form.field.rename}
+            help={
+              <>
+                <p>
+                  {l('You should only use the checkbox above to fix errors (e.g. typos).')}
+                </p>
+                <p>
+                  {exp.l(
+                    `If a name appears on the cover of a release, don't check the box:
```

Comment:
one last nitpick (the other "don’t" in the string uses it so it's inconsistent now :))
```suggestion
                    `If a name appears on the cover of a release, don’t check the box:
```